locales/en-US.json

```diff
@@ -471,5 +471,6 @@
     "crash_page_switch_instance": "tried to <a href=\"`x`\">use another instance</a>",
     "crash_page_read_the_faq": "read the <a href=\"`x`\">Frequently Asked Questions (FAQ)</a>",
     "crash_page_search_issue": "searched for <a href=\"`x`\">existing issues on GitHub</a>",
-    "crash_page_report_issue": "If none of the above helped, please <a href=\"`x`\">open a new issue on GitHub</a> (preferably in English) and include the following text in your message (do NOT translate that text):"
+    "crash_page_report_issue": "If none of the above helped, please <a href=\"`x`\">open a new issue on GitHub</a> (preferably in English) and include the following text in your message (do NOT translate that text):",
+    "video_not_exist_in_playlist": "The video requested doesn't exist in the playlist. Click  <a href=\"`x`\">here</a> for the playlist home page."
```

Comment:
```suggestion
    "error_video_not_in_playlist": "The requested video doesn't exist in this playlist. <a href=\"`x`\">Click  here for the playlist home page.</a>"
```

Reply:
Changes proposed:
* `error_` prefix in error messages' IDs 
* Minor wording changes
* Links containing non-descriptive text (like "click here") should be avoided, as they aren't accessible => using the whole phrase as the link's text.

Reply:
Done